client/app/components/calendar/layout/SideCalendar.jsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import moment from 'moment';
+import Calendar from 'rc-calendar';
+import bindAll from 'lodash/bindAll';
+import instanceOfMoment from 'proptypes/moment';
+import 'rc-calendar/assets/index.css';
+import './side_calendar.scss';
+
+export default class SideCalendar extends React.Component {
+  static contextTypes = {
+    router: React.PropTypes.object
+  };
+
+  static propTypes = {
+    date: instanceOfMoment.isRequired
+  };
+
+  constructor(...args) {
+    super(...args);
+    this.state = { value: moment(this.props.date) };
+
+    bindAll(this, ['onSelect', 'updateSelectedDate']);
+  }
+
+  onSelect(date) {
+    this._routeToSelectedDate(date);
+    this.updateSelectedDate(date);
+  }
+
+  updateSelectedDate(date) {
+    this.setState({ value: moment(date) });
+  }
+
+  disableWeekend(date) {
```

Comment:
This method doesn't disable weekends. It returns true if the date is a weekday. I'd call it `isWeekday`